src/components/AppliedFilters.tsx

```diff
@@ -58,42 +58,29 @@ export interface AppliedFiltersProps {
  * @returns A React element for the applied filters
  */
 export function AppliedFilters(props: AppliedFiltersProps): JSX.Element {
-  const nlpFilters = useSearchState(state => state.vertical.appliedQueryFilters);
   const isLoading = useSearchState(state => state.searchStatus.isLoading);
-  const hasResults = !!useSearchState(state => state.vertical.results);
-  const filters = useStateUpdatedOnSearch(state => state.filters);
 
   const {
-    hiddenFields,
+    hiddenFields = DEFUALT_HIDDEN_FIELDS,
     customCssClasses = {},
     hierarchicalFacetsDelimiter = DEFAULT_HIERARCHICAL_DELIMITER,
     hierarchicalFacetsFieldIds
   } = props;
 
-  const appliedFilters: GroupedFilters = useMemo(() => {
-    return pruneAppliedFilters(
-      hasResults ? (filters ?? {}) : {},
-      nlpFilters ?? [],
-      hiddenFields ?? ['builtin.entityType'],
-      hierarchicalFacetsFieldIds ?? [],
-      hierarchicalFacetsDelimiter
-    );
-  }, [
-    hasResults,
-    filters,
-    hiddenFields,
-    hierarchicalFacetsDelimiter,
-    hierarchicalFacetsFieldIds,
-    nlpFilters
-  ]);
+  const removableFilters = useRemovableFilters(
```

Comment:
The verb `use` seems a bit funny here and below. We're not using anything really, we're getting the values. Do these need to be React Hooks or can they just be normal functions? The benefit of writing your own Hook is that you can share stateful logic between Components. Are we doing that with these functions?